docs/security/authorization.md

```diff
@@ -8,38 +8,38 @@
 
 <h2 id="1">Authorization Modes</h2>
 
-Three primary modes for Kyuubi authorization are available by [spark-authorizer](https://github.com/yaooqinn/spark-authorizer):
+Three primary modes for Kyuubi authorization are available by [spark-authorizer](https://github.com/NetEase/spark-authorizer):
 
 <h4 id="1.1">Storage-Based Authorization</h4>
 
 Enabling Storage Based Authorization in the `Hive Metastore Server` uses the HDFS permissions to act as the main source for verification and allows for consistent data and metadata authorization policy. This allows control over metadata access by verifying if the user has permission to access corresponding directories on the HDFS. Similar with `HiveServer2`, files and directories will be translated into hive metadata objects, such as dbs, tables, partitions, and be protected from end user's queries through Kyuubi.
 
 Storage-Based Authorization offers users with Database, Table and Partition-level coarse-gained access control.
 
-Please refer to the [Storage-Based Authorization Guide](https://yaooqinn.github.io/spark-authorizer/docs/storage_based_authorization.html) in the online documentation for an overview on how to configure Storage-Based Authorization for Kyuubi.
+Please refer to the [Storage-Based Authorization Guide](https://NetEase.github.io/spark-authorizer/docs/storage_based_authorization.html) in the online documentation for an overview on how to configure Storage-Based Authorization for Kyuubi.
```

Comment:
https://NetEase.github.io/spark-authorizer/docs/storage_based_authorization.html does not work.

Should we replace the repo name for spark-authorizer also?

Reply:
@yaooqinn

Reply:
use Submarine Spark security instead？

Reply:
ok

Reply:
But for the Storage-Based Authorization Guide,

There is no related doc in submarine.
https://submarine.apache.org/docs/userDocs/submarine-security/spark-security/README

Reply:
Just remove that line?